src/nvim/lua/base64.c

```diff
@@ -18,13 +18,19 @@ static int nlua_base64_encode(lua_State *L)
   }
 
   if (lua_type(L, 1) != LUA_TSTRING) {
-    luaL_argerror(L, 1, "expected string");
+    luaL_argerror(L, 1, "Expected string");
```

Comment:
The casing here is intentional, as this error message is shown as a substring of a longer error message. Try `:lua =vim.base64.encode(1)` and it will show `bad argument #1 to 'encode' (expected string)`.